.github/workflows/test-cf-worker-next.yml

```diff
@@ -0,0 +1,73 @@
+name: "Test edge function compatibility for Next.js on Cloudflare Workers"
+on: push
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        env:
+            CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+            CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+            APP_URL: supertokens-node-b95.pages.dev
```

Comment:
I think this should also come from the env, otherwise we'll have to update this again after merging to use our own credentials.